src/pages/group.tsx

```diff
@@ -0,0 +1,28 @@
+import { api } from "~/utils/api";
+
+export default function Group() {
+
+    const getGroup = api.group.getGroup.useQuery({ groupId: "42" });
+    const getDraw = api.draw.getDraw.useQuery({ passphrase: "sara-loves-christmas" });
+  
+    console.log(getGroup.data ? getGroup.data.group : 'no data');
+    const groupData = getGroup.data ? getGroup.data.group : null;
+    let listItems;
+    if (groupData) {
+        listItems = groupData.map(group => 
+                <tr>
+                    <th scope="row">Giver</th><td>{group.giver}</td>
+                    <th scope="row">Passphrase</th><td>{group.passphrase}</td>
+                </tr>
+        );
+    }
+    return (
+        <>
+                <h1>Group</h1>
+                <table>
```

Comment:
Getting this error regarding table formating
<img width="548" alt="Screenshot 2023-03-02 at 11 56 47" src="https://user-images.githubusercontent.com/1824501/222409568-221b3a3e-ef84-44bb-bb4b-9506e0d45f55.png">